src/utils/Processing.ts

```diff
@@ -40,6 +41,19 @@ export async function getResources(
   return resources;
 }
 
+export async function getConfig(
+  inDir: string,
+  defs: fhirdefs.FHIRDefinitions,
+  externalDeps: string[]
+): Promise<ExportableConfiguration> {
+  const lake = getLakeOfFHIR(inDir);
```

Comment:
It seems that now we're calling `getLakeOfFHIR` twice; once here and once in `getResources`.  Since that is a fairly expensive operation, and since it can also cause log messages (which will now be duplicated), I think we should refactor the code so that we only do it once and `getConfig` and `getResources` can both use the same lake of FHIR.  In the end, this is all to just build up the `FHIRProcessor`, so maybe that is the part that can be refactored out and shared.